src/middlewared/middlewared/plugins/system_advanced/gpu.py

```diff
@@ -9,22 +8,6 @@ class Config:
         namespace = 'system.advanced'
         cli_namespace = 'system.advanced'
 
-    @accepts(roles=['SYSTEM_ADVANCED_READ'])
-    @returns(Dict(additional_attrs=True))
-    def get_gpu_pci_choices(self):
```

Comment:
This is being used by UI as well iirc, @Qubad786 can you confirm ?

Reply:
It seems UI is using `device.get_info` right now but it should be using this endpoint instead and i think we already have a ticket for UI to use this endpoint

Reply:
Ticket https://ixsystems.atlassian.net/browse/NAS-130983